[DOC-1347][2025.2.4][2026.1][ycm] Add flags for using TS response cache in auth in CM (#31783)
yugabyte/yugabyte-db: docs/content/stable/additional-features/connection-manager-ysql/ycm-setup.md

| ysql_conn_mgr_server_lifetime | The maximum duration (in seconds) that a backend PostgreSQL connection managed by YSQL Connection Manager can remain open after creation.<br>Default: 3600 |
| ysql_conn_mgr_log_settings | Comma-separated list of log settings for YSQL Connection Manger. Can include 'log_debug', 'log_config', 'log_session', 'log_query', and 'log_stats'.<br>Default: "" |
| ysql_conn_mgr_use_auth_backend | When set to true ("auth backend mode"), each incoming authentication request is handled by a freshly spawned postgres backend. Set this parameter to false to use an authentication passthrough mechanism, where backends from a pool of "control backends" are reused for all authentication requests; this can potentially reduce new connection acquisition latency. Contact {{% support-general %}} about whether setting this parameter to false is recommended for your workload.<br>Default: true |

> | ysql_enable_read_request_cache_for_connection_auth | Uses the YB-TServer response cache during connection authentication to prefetch authorization catalog metadata (for example, role and database ACL information), reducing YB-Master RPC overhead and improving connection acquisition latency when YSQL Connection Manager is enabled. Requires `ysql_enable_read_request_caching` to be enabled (default: true). Not supported when login profiles are enabled (`ysql_enable_profile`). Auth cache entries are refreshed according to `pg_cache_response_trust_auth_lifetime_limit_ms` (default: 60000 ms).<br>Default: false|

Reviewer: ```suggestion
| ysql_enable_read_request_cache_for_connection_auth | Uses the YB-TServer response cache during connection authentication to prefetch authorization catalog metadata (for example, role and database ACL information), reducing YB-Master RPC overhead and improving connection acquisition latency when YSQL Connection Manager is enabled. Requires [ysql_enable_read_request_caching](../../../reference/configuration/yb-tserver/#ysql-enable-read-request-caching) which is enabled by default. Not supported when login profiles are enabled ([ysql_enable_profile](../../../reference/configuration/yb-tserver/#ysql-enable-profile)). Auth cache entries are refreshed according to `pg_cache_response_trust_auth_lifetime_limit_ms` (default: 60000 ms).<br>Default: false |
```
